lib/stripe/config.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Stripe.Config do
+  @moduledoc """
+  Utility that handles interaction with the application's configuration
+  """
+
+  @base_key :stripity_stripe
+
+  @doc """
+  Resolves the given key from the application's configuration returning the
+  wrapped expanded value. If the value was a function it get's evaluated, if
+  the value vas a touple of three elements it gets applied.
+  """
+  @spec resolve(any) :: any
```

Comment:
Could this be `atom` instead of `any`?

Reply:
I changed this to be an `atom` and added a catch all that raises if the `key` isn't an `atom`